src/components/Game.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game: React.FC = () => {
+  const initialBoard = ['', '', '', '', '', '', '', '', '']
+  const [history, setHistory] = useState<string[][]>([initialBoard])
+  const [isXNext, setIsXNext] = useState<boolean>(true)
+  const status = 'Next Player: ' + (isXNext ? 'X' : 'O')
+
+  /**
+   * based on the range value, sets the game to history[n]
+   * @param n
+   */
+  function jumpTo(n: number) {
+    setIsXNext(n % 2 === 0)
+    const newHist = history.slice(0, n + 1)
+    setHistory(newHist)
+  }
+
+  /**
+   * pointer function for handling click events from square component
+   * adds current board state to history[] state
+   * @param squares
+   */
+  function update(squares: string[]) {
```

Comment:
```suggestion
  function addGameHistory(squares: string[]) {
```

just like before, I'd be explicit with the name since you're passing it around so much. It's also not an update function really, it just appends more game history each call.